### PR TITLE
Can't submit OSIRIS run with manual_submission

### LINKED ISSUE #884: Can't submit OSIRIS run with manual_submission
Issue raised by: developer

### What?
Manual submission fails with

```
python manual_submission.py OSIRIS 144378
```

because ICAT prefix is OSI however *.nxs files for this instrument have format OSIRIS00144378.nxs

extend script to handle this use case also.

### PR DESCRIPTION
**Pull request by Anders-Markvardsen**
### Summary of work
Enable OSIRIS to work with manual_submission.

Also tied up doc of method changed in recent PR.

### How to test your work
* test that can submit e.g. `python manual_submission.py OSIRIS 144376`
* test readability

### Additional comments
Not sure how easy it is to regression test this with the current mocking setup. Hence left this as is, but ideas welcome

Fixes #884

### CODE REVIEW

File: scripts/manual_operations/manual_submission.py [hunk 1]
```diff
@@ -87,12 +87,30 @@ def get_location_and_rb_from_icat(icat_client, instrument, run_number, file_ext)
         print("ICAT not connected")  # pragma: no cover
         sys.exit(1)  # pragma: no cover
 
-    icat_instrument_prefix = get_icat_instrument_prefix(instrument)
-    file_name = f"{icat_instrument_prefix}{str(run_number).zfill(5)}.{file_ext}"
+    # look for file-name assuming file-name uses full instrument name
+    file_name = f"{instrument}{str(run_number).zfill(5)}.{file_ext}"
     datafile = icat_client.execute_query("SELECT df FROM Datafile df WHERE df.name = '"
                                          + file_name +
                                          "' INCLUDE df.dataset AS ds, ds.investigation")
```
Comment (JEThompson97) [inline scripts/manual_operations/manual_submission.py:94]: Given we use this same query 4 times by my count, I wonder if it'd be worth refactoring.
The first idea that comes to mind would be a simple method:
```python
def create_query(file_name):
    return f"SELECT df FROM Datafile df WHERE df.name = '{file_name}' INCLUDE df.dataset AS ds, ds.investigation"
```

**Review by JEThompson97 — APPROVED** [review]

File: scripts/manual_operations/manual_submission.py (hunk 1 above)
Comment (Anders-Markvardsen) [reply scripts/manual_operations/manual_submission.py]: thanks and implemented and with a few smaller updates to tests also

**Comment by Anders-Markvardsen** [comment]
I have taken the backport-PR label off since this fix does not concerns that system in operation, and left it as optional to backport into release.

**Comment by dtasev** [comment]
Looks good

**Pull request merged**